Assets/PatchKit Patcher/Scripts/AppData/Local/TemporaryDirectory.cs

```diff
@@ -50,57 +44,23 @@ public string GetUniquePath()
             return uniquePath;
```

Comment:
What should happen if temporary file is not created after 1000 tries? Maybe throw an exception?